content/id/docs/concepts/containers/images.md

```diff
@@ -0,0 +1,370 @@
+
+## Menggunakan Registri Privat (_Private Registry_)
+
+Biasanya kita memerlukan _key_ untuk membaca _image-image_ yang tersedia pada suatu registri privat.
+Kredensial ini dapat disediakan melalui beberapa cara:
+
+  - Menggunakan Google Container Registry
+    - per-klaster
+    - konfigurasi secara otomatis pada Google Compute Engine atau Google Kubernetes Engine
+    - semua Pod dapat membaca registri privat yang ada di dalam proyek
+  - Menggunakan Amazon Elastic Container Registry (ECR)
+    - menggunakan IAM _role_ dan _policy_ untuk mengontrol akses ke repositori ECR
+    - secara otomatis _refresh_ kredensial login ECR
+  - Menggunakan Oracle Cloud Infrastructure Registry (OCIR)
+    - menggunakan IAM _role_ dan _policy_ untuk mengontrol akses ke repositori OCIR
+  - Menggunakan Azure Container Registry (ACR)
+  - Menggunakan IBM Cloud Container Registry
+    - menggunakan IAM _role_ dan _policy_ untuk memberikan akses ke IBM Cloud Container Registry
+  - Konfigurasi Node untuk otentikasi registri privat
+    - semua Pod dapat membaca registri privat manapun
+    - memerlukan konfigurasi Node oleh admin klaster
+  - Pra-unduh _image_
+    - semua Pod dapat menggunakan _image_ apapun yang di-_cached_ di dalam sebuah Node
+    - memerlukan akses root ke dalam semua Node untuk pengaturannya
+  - Mengatur ImagePullSecrets dalam sebuah Pod
+    - hanya Pod-Pod yang menyediakan _key_ sendiri yang dapat mengakses registri privat
+
+Masing-masing opsi dijelaskan lebih lanjut di bawah ini.
+
+### Menggunakan Google Container Registry
+
+Kubernetes memiliki dukungan _native_ untuk [Google Container
+Registry (GCR)](https://cloud.google.com/tools/container-registry/), ketika dijalankan pada
+Google Compute Engine (GCE). Jika kamu menjalankan klaster pada GCE atau Google Kubernetes Engine,
+cukup gunakan nama panjang _image_ (misalnya gcr.io/my_project/image:tag).
+
+Semua Pod di dalam klaster akan memiliki akses baca _image_ di registri ini.
+
+Kubelet akan melakukan otentikasi GCR menggunakan _service account_ yang dimiliki
+_instance_ Google. _Service acccount_ pada _instance_ akan memiliki sebuah `https://www.googleapis.com/auth/devstorage.read_only`,
+sehingga dapat mengunduh dari GCR di proyek yang sama, tapi tidak untuk unggah.
+
+### Menggunakan Amazon Elastic Container Registry
+
+Kubernetes memiliki dukungan _native_ untuk [Amazon Elastic Container Registry](https://aws.amazon.com/ecr/), ketika Node adalah
+AWS EC2 _instance_.
+
+Cukup gunakan nama panjang _image_ (misalnya `ACCOUNT.dkr.ecr.REGION.amazonaws.com/imagename:tag`) di dalam definisi Pod.
+
+Semua pengguna klaster yang dapat membuat Pod akan bisa menjalankan Pod yang dapat menggunakan
+_image-image_ di dalam registri ECR.
+
+Kubelet akan mengambil dan secara periodik me-_refresh_ kredensial ECR, yang memerlukan _permission_ sebagai berikut:
```

Comment:
```suggestion
Kubelet akan mengambil dan secara periodik memperbarui kredensial ECR, yang memerlukan _permission_ sebagai berikut:
```

super nit, might be ignored

Reply:
fixed in https://github.com/kubernetes/website/pull/19784/commits/3fb85d866101c08ce75ea0265beac4929e73f29c